regression/library/test_manycore_vcache_sequence.c

```diff
@@ -112,6 +112,56 @@ int test_manycore_vcache_sequence() {
                         
                         mismatch = mismatch || !data_match;
                 }
+
+
+                // Write a 4-byte word into mis-aligned addresses to 
+                // verify that API returns an invalid address error
+                hb_mc_npa_t npa = hb_mc_npa_from_x_y(dram_coord_x,
+                                                     dram_coord_y,
+                                                     BASE_ADDR+ 1);
+                write_data = rand();
+                err = hb_mc_manycore_write_mem(mc, &npa, &write_data, sizeof(write_data));
+                if (err != HB_MC_SUCCESS) { 
+                        bsg_pr_test_info("%s: Expected an invalid address error. We're good!\n", __func__);
+                        bsg_pr_err("%s: Did not catch write into invalid mis-alined address: 0x%08" PRIx32 ".\n",
+                                   __func__, hb_mc_npa_get_epa(&npa));
+
+                }
+                else {
+                        bsg_pr_err("%s: Did not catch write into invalid mis-alined address: 0x%08" PRIx32 ".\n",
+                                   __func__, npa);
```

Comment:
You want `hb_mc_npa_get_epa(&npa)` here.

Reply:
*sigh* if only stdio were type-safe...

Reply:
separate issue, we should file an issue to turn on -Wall -Werror in the regression tests.

Reply:
/giphy can it. 

(My magic abilities only extend so far)

Reply:
Yes, to -wall/-werror.

Reply:
I think I did at some point. I think the loader test broke

Reply:
The underlying issue here is fixed

Reply:
Turning on -wall -werror is never the trivial task it should be...